src/pages/DetailsPage.js

```diff
@@ -66,6 +67,10 @@ const getPhoneNumber = (details) => {
 
 const DetailsPage = (props) => {
     const details = props.personalDetails[props.route.params.login];
```

Comment:
I think that this line is probably what is causing this crash: https://github.com/Expensify/App/issues/9545

It doesn't seem like a very safe way to access properties. It appears like there can be cases where `props.route.params` is `undefined`. Is that a valid case? Should this use `lodashGet` instead?

Reply:
I think you are right. If you just open `https://new.expensify.com/details` without the param `login`, it crashes. I'll spin a small PR to fix it.

Reply:
Also, this feels low priority, I don't know how a user ended up with the URL without the parameter.